File: WebApplication/src/modules/application/models/application-file.dto.ts
import { AttachedFile } from 'src/infrastructure/models/attached-file.model';
import { ApplicationFileTypeNomenclatureDto } from './application-file-nomenclature.dto';
export class ApplicationFileDto {
  id: number;
  type: ApplicationFileTypeNomenclatureDto;
  attachedFile: AttachedFile;
  fileDescription: string;
}

File: VisaD.Application/Applications/Dtos/ApplicationFileDto.cs
using FileStorageNetCore.Models;
using VisaD.Application.Nomenclatures.Dtos;

namespace VisaD.Application.Applications.Dtos
{
	public class ApplicationFileDto
	{
		public int Id { get; set; }

		public ApplicationFileTypeNomenclatureDto Type { get; set; }

		public AttachedFile AttachedFile { get; set; }

		public string FileDescription { get; set; }
	}
}
